src/lib/browser.js

```diff
@@ -1,25 +1,52 @@
 const puppeteer = require("puppeteer");
-const client = require("./client");
-const { reset } = require("../utils/color");
 
-module.exports = {
-  launch: async (options) => {
-    const browser = await puppeteer.launch({
+function createBrowserInstance() {
+  let browser = null;
+  let page = null;
+
+  const launch = async (options) => {
+    const browserLaunched = await puppeteer.launch({
       headless: true,
       args: ["--no-sandbox"],
       ...options,
     });
 
-    console.log(reset, "[!] Initializing scan...");
+    const pageCreated = await browserLaunched.newPage();
+
+    browser = browserLaunched;
+    page = pageCreated;
+  };
+
+  const navigateToUrl = async (
+    url,
+    options = {
+      waitUntil: "load",
+      timeout: 0,
+    }
+  ) => await page.goto(url, options);
```

Comment:
Redundant use of `await` on a return value.

Reply:
Done.